poster/poster.tex

```diff
@@ -70,7 +70,7 @@
             
 \begin{block}{Introduction}
 	\begin{itemize}
-		\item Janus is an imperative reversible programming language, meaning that every computation and function can be reversed. \textbf{Hanus} is an extended implementation of Janus that can be compiled straight to Haskell. Because of this, Hanus contains many awesome Haskell features!
```

Comment:
Maybe split it to a second `\item` in `\textbf{Hanus}`?

Reply:
Agreed.